### ISSUE TITLE
Scrolling broken with large number of pods 

### ISSUE THREAD

**Issue by nodesocket** [kubetail-org/kubetail#46]
When expanding a large list of deployments or pods I'm unable to scroll the browser window in Google Chrome to see all items. Just a guess, but could be css related.

![Screenshot 2024-04-01 at 11 52 24 PM](https://github.com/kubetail-org/kubetail/assets/523312/4e1cdb84-fe91-4a72-8dc3-f34edef31186)

**Comment by amorey** [comment]
Thanks for the heads up! I just noticed it myself too right before seeing this. Will fix asap.

**Comment by amorey** [comment]
This should be fixed in the latest version (0.4.3):

```
helm repo update kubetail
helm upgrade kubetail kubetail/kubetail --namespace kubetail
```

Thanks again for the bug report. Let me know if it isn't working as expected.
